build_dcos_launch.sh

```diff
@@ -1,32 +1,19 @@
 #!/bin/bash
 #
 # Simple helper script to build dcos-launch binary
-# NOTE: this needs to be kept in sync with gen.build_deploy.bash::make_dcos_launch()
-
+# Note: must be run after this repo has been prepped with prep_local or prep_teamcity
 set -x
 set -o errexit -o pipefail
-
-# Cleanup from previous build
-rm -rf /tmp/dcos_build_venv
-
 # Force Python stdout/err to be unbuffered.
 export PYTHONUNBUFFERED="notemtpy"
-
-# Create a python virtual environment to install the DC/OS tools to
-python3 -m venv /tmp/dcos_launch_venv
-. /tmp/dcos_launch_venv/bin/activate
-
-# Make a clean as possible clone
-if [[ -n $(git -C $PWD status --porcelain -uno -z) ]]; then
-  echo "Commit all changes before attempting to build!";
-  exit 1;
-fi
-rm -rf /tmp/dcos-installer-build
-git clone -q "file://$PWD" /tmp/dcos-installer-build/
-pushd /tmp/dcos-installer-build
-# Install the DC/OS tools
-pip install -e /tmp/dcos-installer-build
-cp launch/dcos-launch.spec ./
-pyinstaller --log-level=DEBUG dcos-launch.spec
+pushd packages/dcos-launch
+new_package_name=`mkpanda | tail -n 1 | cut -d':' -f2`
```

Comment:
To be honest, I feel this entire file should be removed. It was only updated because a helper script like this already exists. In truth, this will not be able to build at all on the first build because mkpanda will not evaluate any other part of the tree. Alternatively, you do not want to check every part of the tree when you are just developing dcos-launch. Rather, this should be a Makefile so you can do make/make deps/make launch-deps/make installer/make clean